mediated_transfer.rst

```diff
@@ -0,0 +1,71 @@
+Mediated transfers
+##################
+
+Overview
+========
+
+The protocol supports mediated transfers. Nodes may use them to send payments.
+A :term:`Mediated transfer` may be cancelled and can expire until the initiator reveals the secret.
+
+A mediated transfer is done in two stages, possibly on a series of channels:
+
+- Reserve token :term:`capacity` for a given payment, using a :ref:`locked transfer message <locked-transfer-message>`.
+- Use the reserved token amount to complete payments, using the :ref:`unlock message <unlock-message>`
+
+Mediated Transfers
+==================
+
+A :term:`Mediated Transfer` is a hash-time-locked transfer. Currently raiden supports only one type of lock. The lock has an amount that is being transferred, a :term:`secrethash` used to verify the secret that unlocks it, and a :term:`lock expiration` to determine its validity.
+
+Mediated transfers have an :term:`initiator` and a :term:`target` and a number of mediators in between. The number of mediators can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of mediators, a mediated transfer will require ``10N + 16`` messages to complete. These are:
+
+- ``N + 1`` :term:`locked transfer` or :term:`refund transfer` messages
+- ``1`` :term:`secret request`
+- ``N + 2`` :term:`reveal secret`
+- ``N + 1`` :term:`unlock`
+- ``2N + 3`` processed (one for everything above)
+- ``5N + 8`` delivered
+
+For the simplest Alice - Bob example:
```

Comment:
can we create an example, maybe even consistent with the example in messaging.rst, that has a mediator?